index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    ::before
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+    <p id="discription" class="discription text-center">Thank you for taking the time to help us improve the platform</p>
+      </header>
+    
+    <form id="form">
+      <div class="form-group">
+        <label for="name">Name</label>
+      <input type="text" name="name" placeholder="Enter your name" class="form-control" required/> 
+      </label>
+      </div>
+
+      <div class="form-group">
+        <label for="email">Email</label>
+      <input type="text" name="name" placeholder="Enter your name" class="form-control" required/> 
+      </label>
+      </div>
+
+      
+
+      <div class="form-group">
+        <label for="name">Age(optional)</label>
+      <input type="text" name="name" placeholder="Enter your name" class="form-control" required/> 
+      </label>
+      </div>
+
+      <div class="form-group">
+        <p>Which option best describe your current role?</option></p>
+        <select>
+          <option>select your current role</option>
+          <option value="student">student</option>
+          <option value="Full time job">Full time job</option>
+          <option value="full time learner">Full time learner</option>
+          <option value="prefer not to say">prefer not to say</option>
+          <option value="other">other</option>
+          
+        </select>
+        
+      </div>
+
+
+      <div class="form-group">
+        <p>Would you recommend freeCodeCamp to a friend?</p>
+           <label>
+        <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+              Definetely
+            </label>
+              
+            <label>
+              <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+                    Maybe
+                  </label>
+
+                  <label>
+                    <input name= "user-recommend" type="radio" value="Definetely" class="input-radio" checked />
+                          Not sure
+                        </label>
+
+
+      </div>
+
+      <div class="form-group">
+        <p>What is your favorite feature of freeCodeCamp?</p>
+        <select>
+          <option>select an option</option>
+          <option>Challenges</option>
+          <option>Projects</option>
+          <option>Community</option>
+          <option>Open source</option>
+        </select>
+
+      </div>
```

Comment:
Please use good HTML semantic tags, such as sections, nav, and more as these are good for SEO and can make your code more readable and understandable.This will help you not to use div always